src/applications/letters/components/DownloadLetterLink.jsx

```diff
@@ -38,7 +38,7 @@ export class DownloadLetterLink extends React.Component {
         buttonDisabled = true;
         break;
       case DOWNLOAD_STATUSES.success:
-        buttonText = 'Download letter';
+        buttonText = `${this.props.letterName} (PDF)`;
```

Comment:
Is this always a PDF?  Or is there a way to dynamically tell what the type of the file is?  I'm not sure what types are even supported, maybe we only ever receive PDFs and this is fine to have hard coded.  🤔

Reply:
Fair enough! ✅